libs/accounts/recovery-phone/src/lib/recovery-phone.types.ts

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export interface RecoveryPhone {
+  uid: Buffer;
+  phoneNumber: string;
+  createdAt: number;
+  lastConfirmed: number;
+  lookupData: string;
```

Comment:
What is lookup data?

Reply:
Looks like it information if the phone number is fradulent or active and can recieve message.